_posts/zh/newsletters/2020-08-05-newsletter.md

```diff
@@ -0,0 +1,143 @@
+
+- [Bitcoin Core #19569][] 允许 Bitcoin Core 从使用 wtxid 中继交易的对等方获取 _孤儿_ 交易的父交易。孤儿交易是指从一个对等方接收到的未确认交易，但我们尚未获得其父交易，既不在我们的最佳区块链中，也不在内存池中。更准确地说，孤儿交易至少有一个交易输入，其关联的输出不在 UTXO 集合或我们内存池的 outpoint 映射中。
+
+  当我们收到一个孤儿交易时，我们将其放在一个称为孤儿集的临时数据结构中。然后我们要求发送孤儿交易的对等方也发送我们尚未拥有的父交易。我们可以这样做是因为孤儿交易包含其父交易的 txid。我们只需向对等方发送一个包含这些 txid 的 `getdata` 消息来请求父交易。
+
+  对于 [wtxid 中继对等方][news108 wtxid relay]，交易使用 _wtxid_ 而非 _txid_ 来通告和请求。然而，孤儿交易包含的是其父交易的 txid 而非 wtxid，因此无法使用 wtxid 请求父交易。上周合并的 [PR #18044][Bitcoin Core #18044] 引入了 wtxid 中继对等方，但并未允许从 wtxid 对等方获取父交易。该后续 PR 允许我们使用 txid 来获取这些父交易。
+
+  使用 txid 获取父交易可能最终会被[包中继][topic package relay]机制取代，通过该机制我们可以直接向对等方请求所有未确认的祖先交易。
+
+- [Eclair #1491][] 增加了部分支持创建、使用和关闭使用[锚定输出][topic anchor outputs]的通道，以在正常情况下减少链上交易费用，并在必要时提高费用以确保安全。此实现处理了所有基本内容，但尚不支持双边通道关闭或实际的费用提升；这些将稍后添加。此外，与 LND 实现的互操作性测试揭示了一个需要在[规范][anchor spec discuss]中澄清的情况。
+
+- [LND #4488][] 更新了用户可以设置的最小 CLTV 过期差值至 18 个区块，以符合更新的[推荐值][BOLTs #785]。默认值仍为 40 个区块。当 LN 支付的结算截止时间仅剩这些区块时，LND 将单方面关闭通道以确保最新状态记录在链上。然而，过期差值越高，支付在通道中暂时卡住的时间就越长（无论是意外还是故意）。这导致一些 LN 实现使用优化低 CLTV 过期差值的路由算法，进而导致一些用户设置了特别不安全的差值。这个新的最小值应当有助于防止没有经验的用户轻率地设置不安全的值。
+
+- [BIPs #948][] 更新了 [BIP174][] 规范中的 [PSBT][topic psbt] 输入记录，明确允许为单个输入同时提供非见证 UTXO（完整交易）和见证 UTXO。这与 Bitcoin Core 中的[当前行为][news105 fee overpayment]一致，其动机是防止在未包含非见证 UTXO 的多输入 segwit PSBT 上发生的多付费用攻击（详见[之前的 Newsletter][fee overpayment attack]）。
+
+- [BIPs #947][] 更新了 [BIP325][] 规范中的 [signet][topic signet]，更改了区块签名验证方法。Signet 是一个测试网络，其中有效区块由受信任的签名者签署，这一变化消除了某些问题并简化了某些类型的测试。
+
+  之前，signet 假定使用与传统比特币脚本兼容的签名（例如 DER 编码的 ECDSA 签名）。此更改后，signet 改为使用一对虚拟交易——这些交易在区块链上无效，也不会被包含在区块中，但比特币软件可以轻松构建（直接或使用 [PSBT][topic PSBT]）。第一个交易承诺支付给网络的受信任签名者脚本。第二个虚拟交易随后花费第一个虚拟交易的输出。第二个虚拟交易的签名被包含在区块的 coinbase 交易中，以证明区块已被有效签署。
+
+  这种新方法的主要优势在于它允许使用 segwit 交易。当前 segwit v0 中可用的操作码几乎与传统脚本完全相同 <!-- 我认为唯一的变化是 OP_CODESEPARATOR -->，因此这看起来可能无关紧要——但如果 segwit v1（[taproot][topic taproot]）在 signet 上可用，这将允许使用 [schnorr 签名][topic schnorr signatures]来签署区块。未来的协议更改可能也会使用 segwit，这将允许这些功能得到使用。次要的优势是，任何能够为任意输入签署 PSBT 的软件或硬件现在都可以作为 signet 的受信任签名者运行。
+
+{% include references.md %}
+{% include linkers/issues.md issues="19569,1491,4488,948,947,785,18044,558,17977" %}
+[C-Lightning 0.9.0]: https://github.com/ElementsProject/lightning/releases/tag/v0.9.0
+[Bitcoin Core 0.20.1]: https://bitcoincore.org/bin/bitcoin-core-0.20.1/
+[LND 0.11.0-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.11.0-beta.rc1
+[fee overpayment attack]: /zh/newsletters/2020/06/10/#fee-overpayment-attack-on-multi-input-segwit-transactions
+[news105 fee overpayment]: /zh/newsletters/2020/07/08/#bitcoin-core-19215
+[news107 notable]: /zh/newsletters/2020/07/22/#值得注意的代码和文档更改
+[anchor spec discuss]: https://github.com/lightningnetwork/lightning-rfc/pull/688#issuecomment-661669232
+[ivgi minsc]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-July/018062.html
```

Comment:
The Linux Foundation mailing list archive links no longer work.

We recently merged updates to all of the links to a new url, see: https://github.com/bitcoinops/bitcoinops.github.io/pull/2018

Can you update this PR's linuxfoundation links and future linuxfoundation links to use the new urls? @bittomhan